applauncher/src/main/java/de/interseroh/tmb/applauncher/client/ApplauncherWebApp.java

```diff
@@ -70,7 +70,7 @@ public void onModuleLoad() {
 
 		RootPanel appLauncherRoot = getWidgets(TMB_APP_LAUNCHER);
 		appLauncherRoot.getElement().addClassName(CSS_BLOCK);
-		appLauncherRoot.getElement().addClassName("hidden-xs");
+		//appLauncherRoot.getElement().addClassName("hidden-xs hidden-sm");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)